packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+rpm2deb() {
+	set -x
+	chown build:build /home/build
+	pushd /debbuild/PKG 1>/dev/null
+	for RPM in /rpmbuild/SRC/calico*.rpm; do
+		alien --bump=0 "$RPM"
+	done
+	cp -p /rpmbuild/SRC/*.deb /debbuild/PKG
+	chown "${TARGET_UID}:${TARGET_GID}" \
+		/debbuild/DEBS/*
+}
+
+builddeb() {
+	#need to be done with user "build"
+	set -x
+	chown build:build /home/build
+        pushd /debbuild/PKG 1>/dev/null
+	for DEBSRC in /debbuild/PKG; do
+		pushd /debbuild/PKG/metalk8s-sosreport/debian
+		debuild -b -uc -us
```

Comment:
Need to be run as build user:
```
su build -c "debuild -b -uc -us"
```

Reply:
Not necessary, i prefer to use a `chown` after which will apply another owner that `root`.